sr_Latn_RS/webapp/src/main/webapp/i18n/all_sr_Latn_RS.properties

```diff
@@ -972,3 +972,31 @@ label.dateTimeWithPrecision.day_capitalized = Dan
+password_reset_complete_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nLozinka je uspešno promenjena.\n\nVaša lozinka povezana sa nalogom ${userAccount.emailAddress} je promenjena.\n\nHvala Vam.
+
+confirm_email_changed_email_html_text = <p>\n Poštovani ${userAccount.firstName} ${userAccount.lastName}:\n </p>\n\n <p>\n Nedavno ste promenili email adresu povezanu sa nalogom za korisnika\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n Hvala Vam.\n </p>
+confirm_email_changed_email_plain_text = Poštovani ${userAccount.firstName} ${userAccount.lastName}:\n\nNedavno ste promenili email adresu povezanu sa nalogom za korisnika\n${userAccount.firstName} ${userAccount.lastName}\n\nHvala Vam.
+
+first_time_external_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Čestitamo!</strong>\n </p>\n\n <p>\n Kreirali smo novi VIVO nalog povezan sa ${userAccount.emailAddress} email adresom.\n </p>\n\n <p>\n Hvala!\n </p>
+first_time_external_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nČestitamo!\n\nKreirali smo novi VIVO nalog povezan sa ${userAccount.emailAddress} email adresom.\n\nHvala!
+
+password_created_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Lozinka je uspešno kreirana.</strong>\n </p>\n\n <p>\n Vaša nova lozinka povezana sa nalogom ${userAccount.emailAddress} je kreirana.\n </p>\n\n <p>\n Hvala Vam.\n </p>
+password_created_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nLozinka je uspešno kreirana.\n\nVaša nova lozinka povezana sa nalogom ${userAccount.emailAddress} je kreirana.\n\nHvala Vam.
+
+application_error_email_html_text = <html>\n <head>\n <title>${subject!}</title>\n </head>\n <body>\n <p>\n ${i18n().error_occurred_at(siteName!,datetime!)}\n </p>\n\n <p>\n <strong>${i18n().requested_url}:</strong> ${requestedUrl!}\n </p>\n\n <p>\n <#if errorMessage?has_content>\n <strong>${i18n().error_message}:</strong> ${errorMessage!}\n </#if>\n </p>\n\n <p>\n <strong>${i18n().stack_trace}</strong> (${i18n().trace_available(siteName!)}):\n <pre>${stackTrace!}</pre>\n </p>\n\n <#if cause?has_content>\n <p><strong>${i18n().caused_by}:</strong>\n <pre>${cause!}</pre>\n </p>\n </#if>\n\n </body>\n</html>
+
+application_error_email_plain_text = ${i18n().error_occurred_at(siteName!,datetime!)}\n\n${i18n().requested_url}: ${requestedUrl!}\n\n<#if errorMessage?has_content>\n ${i18n().error_message}: ${errorMessage!}\n</#if>\n\n${i18n().stack_trace} (${i18n().trace_available(siteName!)}):\n${stackTrace!}\n\n<#if cause?has_content>\n${i18n().caused_by}:\n${cause!}\n</#if>
+
+application_error_email_subject = ${i18n().error_occurred(siteName!)}
```

Comment:
@litvinovg can you please try to improve those two lines in accordance with our discussion: to copy here text from error_occurred translation, and to move this i18n method invocation out from the property file

Reply:
Done